core/src/main/scala/cats/arrow/NaturalTransformation.scala

```diff
@@ -1,6 +1,23 @@
 package cats
 package arrow
 
-trait NaturalTransformation[F[_], G[_]] extends Serializable {
+trait NaturalTransformation[F[_], G[_]] extends Serializable { outer =>
```

Comment:
Consistency robot here: I think in other cases the self type is called `self`. Do we want to have this consistent across the code base?